lib/active_merchant/billing/gateways/paystation.rb

```diff
@@ -100,101 +100,101 @@ def scrub(transcript)
 
       private
 
-        def new_request
```

Comment:
Did most of this file get re-ordered?

Reply:
No. I'm not honestly entirely clear what you're responding to; the lines match up one-for-one.

Reply:
Oh, I see, GitHub's being dumb in side-by-side mode. Either turn on "ignore whitespace" or use a different diff viewer. Or you can look at the actual line numbers.